### PR TITLE
feature/enable-null-urls

### PR DESCRIPTION
**Pull request by fivetran-catfritz**
**Are you a current Fivetran customer?** 

Fivetran created PR. 
**What change(s) does this PR introduce?** 

- For use in the [dbt_ad_reporting package](https://github.com/fivetran/dbt_ad_reporting), users can now allow records having nulls in url fields to be included in the `ad_reporting__url_report` model.
- Disabled the `not_null` test for `pinterest_ads__url_report` when null urls are allowed

**Did you update the CHANGELOG?** 

- [x] Yes

**Does this PR introduce a breaking change?**

- [ ] Yes (please provide breaking change details below.)
- [x] No  (please provide an explanation as to how the change is non-breaking below.)

**Did you update the dbt_project.yml files with the version upgrade (please leverage standard semantic versioning)? (In both your main project and integration_tests)** 

- [x] Yes

**Is this PR in response to a previously created Bug or Feature Request**

- [x] Yes, Issue/Feature [#59](https://github.com/fivetran/dbt_ad_reporting/issues/59)
- [ ] No 

**How did you test the PR changes?** 

 

- [x] BuildKite  
- [x] Local (please provide additional testing details below)

**Select which warehouse(s) were used to test the PR**
 

- [x] BigQuery
- [x] Redshift
- [x] Snowflake
- [x] Postgres
- [x] Databricks
- [ ] Other (provide details below)

**Provide an emoji that best describes your current mood**
 
🍝 

**Feedback**

We are so excited you decided to contribute to the Fivetran community dbt package! We continue to work to improve the packages and would greatly appreciate your [feedback](https://www.surveymonkey.com/r/DQ7K7WW) on our existing dbt packages or what you'd like to see next.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,15 @@
+## 🎉 Features 🎉
+- Added ability for a user to allow records having nulls in url fields to be included in the `ad_reporting__url_report` and/or the underlying `...url_report` models. This is done by setting the below variable to `False` in your `dbt_project.yml` file.
```
Comment (fivetran-sheringuyen) [inline CHANGELOG.md:2]: you can use the wildcard asterisk to denote all url reports:

`*url_report`

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,15 @@
...
+```yml
+vars:
+  ad_reporting__url_report__using_null_filter: False # Use this variable to allow null urls for ALL Fivetran ad packages included in your project. Default is True. 
+```
+- Updated README with this information. 
+
+## 🚘 Under the Hood 🚘
+- Disabled the `not_null` test for `ad_reporting__url_report` when null urls are allowed. [#72](https://github.com/fivetran/dbt_ad_reporting/pull/72)
+
+
```
Comment (fivetran-sheringuyen) [inline CHANGELOG.md:12]: remember to move the above section into the migrations changelog entry!

File: CHANGELOG.md (hunk 1 above)
Comment (fivetran-sheringuyen) [reply CHANGELOG.md:2]: and also, remember to cite the PR here!

File: packages.yml [hunk 3]
```diff
@@ -8,47 +8,47 @@ packages:
   # - package: fivetran/snapchat_ads
   #   version: [">=0.5.0", "<0.6.0"]
     - git: https://github.com/fivetran/dbt_snapchat_ads.git
-      revision: MagicBot/dbt-utils-cross-db-migration
+      revision: feature/enable-null-urls
```
Comment (fivetran-sheringuyen) [inline packages.yml:11]: reminder to switch revisions once all upstream packages have been merged with the migration branch!

**Review by fivetran-sheringuyen — COMMENTED** [review]
hey @fivetran-catfritz, looks mostly good! Just some minor updates before merge. 

Once you are able to get buildkite passing, and docs regenerated, please feel free to merge.

File: CHANGELOG.md (hunk 1 above)
Comment (fivetran-catfritz) [reply CHANGELOG.md:2]: done

File: CHANGELOG.md (hunk 2 above)
Comment (fivetran-catfritz) [reply CHANGELOG.md:12]: done

File: packages.yml (hunk 3 above)
Comment (fivetran-catfritz) [reply packages.yml]: done

**Comment by fivetran-sheringuyen** [comment]
I am going to merge this even though it is still failing databricks. It's the same issue we have been seeing in Google Ads and I believe once that is resolved, this will pass. Oddly, removing it from integration testing is still not completely solving the problem.

**Review by fivetran-sheringuyen — APPROVED** [review]

**Pull request merged**